lib/service/storage/elasticsearch.js

```diff
@@ -1819,7 +1819,7 @@ class ElasticSearch extends Service {
       index: await this._getIndice(index, collection)
     };
 
-    const mutex = new Mutex(`hiddenCollection/delete/${index}`);
+    const mutex = new Mutex(`hiddenCollection/${index}`);
     try {
       await this._client.indices.delete(esRequest);
 
```

Comment:
I see the mutex is acquired after deleting the index in ES, shouldn't the mutex be acquired before ?